### PR TITLE
Use the right certificate for dopplerpages.com

### PR DESCRIPTION
**Pull request by andresmoschini**
Hi @juanasas, @CarlosPintos, @elsupergomez, @cbernat, and team,

This is to configure the right certificate in `dopplerpages.com`, before merging it, I need to add the right certificate in `doppler-swarm-devops` repo and update the file name with the right expiration time.

Could you review it?

TODO:

- [x] Rename the files with the right expiration date.
- [x] Merge MakingSense/doppler-swarm-devops#3 before this

### CODE REVIEW

**Comment by andresmoschini** [comment]
The twin PR is https://github.com/MakingSense/doppler-swarm-devops/pull/3

File: sites-proxy/conf.d/dopplerpages.com.conf [hunk 1]
```diff
@@ -11,6 +11,8 @@ server {
         server_name             dopplerpages.com;
         ssl_certificate         /run/secrets/certificates__fromdoppler.com-20201108.crt;
         ssl_certificate_key     /run/secrets/certificates__fromdoppler.com-20201108.key;
+        ssl_certificate         /run/secrets/certificates__dopplerpages.com-20201108.crt;
```
Comment (cbernat) [inline sites-proxy/conf.d/dopplerpages.com.conf]: Is it ok to have more than one certificate here?

Comment (andresmoschini) [reply sites-proxy/conf.d/dopplerpages.com.conf]: No, sorry, already fixed it in local, thanks!

**Comment by andresmoschini** [comment]
If you want to verify, it is done, I will merge it soon.

**Pull request merged**